docs/instructors/6-codaveri/A-feedback/index.md

```diff
@@ -0,0 +1,24 @@
+---
+slug: /instructor-guide/codaveri/feedback
+title: 6a. Automated Feedback using Codaveri
+sidebar_label: 6a. Automated Feedback
+---
+
+You may use Codaveri for automated grading feedback for coding questions.
+
+First, you will need to enable **Codaveri Evaluation and Feedback** under **Course Settings**.
```

Comment:
This should say "in Course Settings > Components". They wouldn't know where it is otherwise.

Probably a good idea to have a page on Components, then can say if they can't find certain features anywhere, they can check the Components page to see if said component is enabled. Then at least they know that Coursemology features are componentised, and if they don't like certain features, they can choose to disable it. Some instructors will ask us how to disable Forums or some gamification features because they find them intrusive or unnecessary for their course.